### PR TITLE
Update database seeds for 2016

### PR DESCRIPTION
**Pull request by pkoenig10**

### CODE REVIEW

**Comment by purelogiq** [comment]
I looked at this and approve pulling it into s16-67373. If there are any hidden issues we can fix it with another pull requested.

**Comment by minigeek** [comment]
Thanks for cleaning that code up. I kind of realized halfway through that this should be re-used but didn't have the willpower to refactor.

**Comment by pkoenig10** [comment]
No problem, calculating the popularity values manually once was frustrating enough.

**Comment by minigeek** [comment]
I have a google doc that I used to format the inputs and spits out that as
well. I probably should have passed that along…

File: shift_picker.rb [hunk 1]
```diff
@@ -1,132 +1,129 @@
...
+    shift = find_shift(org)
+    assign_shift(org, shift)
+
+    org = (org + 1) % num_orgs
+end
+
+org = 0
+for i in 0..$shifts.length-1
+    $shifts[i].each do |shift|
+        puts "Shift.create({ shift_type: watch_shift, organization: #{orgs[i]}, starts_at: move_on + 21.hours + #{2 * shift}.hours , ends_at: move_on + 21.hours + #{2 * (shift+1)}.hours, required_number_of_participants: 2 })"
```
Comment (minigeek) [inline shift_picker.rb]: We should pull out this as a constant rather than have it hardcoded but buried.

**Pull request merged**